docs/index.md

```diff
@@ -45,17 +45,11 @@ Marquez enables highly flexible [data lineage](https://en.wikipedia.org/wiki/Dat
 Marquez is a modular system and has been designed as a highly scalable, highly extensible platform-agnostic solution for metadata management. It consists of the following system components:
 
 * **Metadata Repository**: Stores all job and dataset metadata, including a complete history of job runs and job-level statistics (i.e. total runs, average runtimes, success/failures, etc).
-* **Metadata API**: RESTful API enabling a diverse set of clients to begin collecting metadata around dataset production and consumption.
+* **Metadata API**: RESTful API enabling a diverse set of clients to begin interacting with metadata around dataset production and consumption.
 * **Metadata UI**: Used for dataset discovery, connecting multiple datasets and exploring their dependency graph.
 
 <br/>
 
-<figure align="center">
-  <img src="./assets/images/design.png">
```

Comment:
I don't know how this image was generated, but it would be great if we can just alter the image to specify that the clients are OpenLineage clients, but then keep the rest.